plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -326,7 +326,7 @@ public Optional<Set<RowId>> loadOrYield()
                         currentPath = createPath(acidInfo, deleteDeltaInfo, sourceFileName);
                         FileSystem fileSystem = hdfsEnvironment.getFileSystem(identity, currentPath, configuration);
                         FileStatus fileStatus = hdfsEnvironment.doAs(identity, () -> fileSystem.getFileStatus(currentPath));
-                        currentPageSource = pageSourceFactory.createPageSource(fileStatus.getPath(), fileStatus.getLen()).orElseGet(() -> new EmptyPageSource());
+                        currentPageSource = pageSourceFactory.createPageSource(fileStatus.getPath().toString(), fileStatus.getLen()).orElseGet(() -> new EmptyPageSource());
```

Comment:
nit: `Path` vs `String` had an advantage that it was a named type, but we can keep strings